databricks/koalas/window.py

```diff
@@ -586,6 +588,12 @@ def _apply_as_series_or_frame(self, func):
             _, name = index_map
             new_index_map.append((SPARK_INDEX_NAME_FORMAT(len(new_index_map)), name))
 
+        if isinstance(kdf.columns, pd.MultiIndex):
+            groupkey = self._groupkeys[0]
```

Comment:
I think it won't cover the cases such as:

```python
pdf.groupby([pdf.a.x, pdf.a.x]).expanding(2).count()
```

Actually, I think we should have a test for that. If you find some time, please work on it :-).

Reply:
@HyukjinKwon just fixed logic & and add test cases